tensorflow/stream_executor/dnn.cc

```diff
@@ -226,6 +226,42 @@ std::tuple<int, int, int> GetDimIndices(const DataLayout& layout,
   return std::make_tuple(depth_idx, batch_idx, spatial_idx);
 }
 
+std::tuple<int, int, int> GetDimIndices(const FilterLayout& layout,
```

Comment:
Could we return a custom struct instead? This way the semantics would be clear.

Reply:
The semantics would become clear, but we will lose consistency with the DataLayout version of this routine here - https://github.com/tensorflow/tensorflow/blob/master/tensorflow/stream_executor/dnn.cc#L193-L194

If you want I can update both routines to use the new custom struct